charters/DataOperations/charter.md

```diff
@@ -0,0 +1,89 @@
+# [Data Operations](mailto:data-operations-team@data.humancellatlas.org)
+
+
+## Description
+
+The Data Coordination Platform (DCP) Data Operations Team is responsible for providing the scientific community with effective tools and resources which maximize the value and use of HCA data. This involves identifying the value added for the HCA consumers from various DCP components and the needs and expectations of the consumers, and connecting these communities through useful, fully vetted products and their distribution.
+
+## Definitions [Optional]
+
+Data submission: a set of raw data derived from the same experimental process, and the metadata describing that process
```

Comment:
language quirk, in DCP terminology we call submission, contribution as the word has more positive connotations. Not super essential but using the same terminology here would be nice

Reply:
Scoured the other charters and _submission_ is clearly defined and used much more frequently, so I have aligned this charter with that (e5224a8)